neo4j/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# integration pip requirements
+time
+requests
+hashlib
+httplib2
```

Comment:
Could we pin httplib2 to a specific version? This well help ensure we are building with the version you have tested and avoid builds breaking later.